ui/DesignSystem/Component/Dropdown/Option.svelte

```diff
@@ -42,18 +53,17 @@
 </style>
 
 <div class="option" on:click={clickHandler} class:selected {style}>
-  {#if variant === 'avatar'}
+  {#if avatarProps}
     <Avatar
       size="small"
       style="overflow:hidden; text-overflow: ellipsis; margin: 0 42px 0 8px;
       --title-color: var(--color-foreground-level-6);"
-      {...avatarProps}
-      {disabled} />
```

Comment:
Where did `{disabled}` go?

Reply:
The `Avatar` component has no `disabled` prop! It never existed in the first place 👻

Reply:
Or if it used to, it no longer does 🤷‍♀️